core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -40,28 +40,73 @@
    *     GenericType) session.execute} call until the result is made available to the client).
    * @param executionProfile the execution profile of this request.
    * @param node the node that returned the successful response.
+   * @deprecated override {@link #onSuccess(Request, long, DriverExecutionProfile, Node, String)}
+   *     instead.
```

Comment:
Can you add at the beginning of the sentence: "This method only exists for backward compatibility. Override ..."

Also, I think we can remove everything except the `@deprecated` tag, to make it more clear that this should not be used.